cdap-distributions/src/packer/cdap-sdk-ubuntu16-with-uri.json

```diff
@@ -172,11 +172,44 @@
     {
       "type": "chef-solo",
       "remote_cookbook_paths": "/var/chef/cookbooks",
-      "run_list": "recipe[openssh],recipe[cdap::sdk]",
+      "run_list": "recipe[openssh]",
+      "skip_install": true,
+      "json": {
+        "openssh": {
+          "server": {
+            "permit_root_login": "without-password",
+            "password_authentication": "no",
+            "use_dns": "no"
+          }
+        }
+      },
+      "except": ["cdap-cloud-sandbox-azure"]
+    },
+    {
+      "type": "chef-solo",
+      "remote_cookbook_paths": "/var/chef/cookbooks",
+      "run_list": "recipe[openssh]",
+      "skip_install": true,
+      "json": {
+        "openssh": {
+          "server": {
+            "client_alive_interval": "120",
```

Comment:
changed it to a global update.  These may have to diverge again at some point, but I agree with keeping them the same as long as we can.